include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -31,117 +31,123 @@
 // DAMAGE.
 //
 // ==========================================================================
-// Author: Hannes Hauswedell <hannes.hauswedell@fu-berlin.de>
-// Author: Marcel Ehrhardt <marcel.ehrhardt@fu-berlin.de>
-// Author: David Heller <david.heller@fu-berlin.de>
-// ==========================================================================
+
+/*!\file
```

Comment:
does not, see https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdfile

> If the file name is omitted (i.e. the line after \file is left blank) then the documentation block that contains the \file command will belong to the file it is located in.

And that is what we want.